lib/DigestSender.php

```diff
@@ -68,6 +69,12 @@ public function sendDigests(int $now): void {
 				// User got todays digest already
 				continue;
 			}
+			$user = $this->userManager->get($user);
```

Comment:
You will need to change Line 88 to this as you're overwriting the user

```
$this->config->setUserValue($user->getUID(), 'activity', 'digest', $timezoneDigestDay[$timezone]);
```

Reply:
Oops sorry about it. Fixed the same.